go/merkletree/merkle_verifier_test.go

```diff
@@ -299,9 +310,30 @@ func getConsistencyProofs() []consistencyTestVector {
+	googleLeaf := new(bytes.Buffer)
+	// Insert leaf cert into local CT instance:
+	// curl -X POST localhost:8088/ct/v1/add-chain -d '{"chain": [...]}'
+	// {"sct_version":0,"id":"3xwuwRUAlFJHqWFoMl3cXHlZ6PfG04j8AC4LvT9012Q=","timestamp":1469742100562,"extensions":"","signature":"BAMARjBEAiBCJXqOLwGlXdh749\/dUiWjp8AlwQUrdg2+PCqEn9lFTAIgfw8o4Q4b0YJQfpemrN4SoBNVZ5IJMAmehk\/TSZexenc="}
+	// Query by hash for the inclusion proof:
+	// curl localhost:8088/ct/v1/get-proof-by-hash?hash=4OEMvu0pQwiIBmNv8x%2B7ivc10k6j0yTuSsmRA3ZPmes%3D&tree_size=1
+	// {"leaf_index":0,"audit_path":[]}
+
+	// Compute leaf value from timestamp
+	if err := ct.SerializeV1SCTMerkleTreeLeafX509(googleLeaf, uint64(1469742100562), googlePEM); err != nil {
```

Comment:
I believe you could use test data to load certificate, SCT from here:
https://github.com/google/certificate-transparency/tree/master/test/testdata
The test will then be much simpler and will only test that SerializeV1SCTMerkleTreeLeafX509 correctly serializes.